overlays/deploy/deployment.yaml

```diff
@@ -17,4 +17,4 @@ spec:
       volumes:
       - name: certs
         secret:
-          secretName: clusterlifecycle-state-metrics-certs
+          secretName: clusterlifecycle-state-metrics
```

Comment:
Where is this secret from?

Reply:
@xuezhaojun it should be created by k8s when creating `service`

Reply:
Got it.